react-frontend/src/components/Items/ShowItem.js

```diff
@@ -57,7 +62,7 @@ class ShowItem extends Component {
                                     <dt>Updated:</dt>
                                     <dd>{this.state.item.updated}</dd>
                                     <dt>Image:</dt>
-                                    <dd>{this.state.item.image}</dd>
+                                    <dd><DisplayImage imgSrc={this.state.img.src}/></dd>
```

Comment:
we can protect against nulls here and images can be optional
```suggestion
                                    <dd> {this.state.img && this.state.img.src ? (<DisplayImage imgSrc={this.state.img.src}/>) : "undefined"}</dd>
```

Reply:
Agreed, optional images. Wouldn't want the whole thing to bug out for something that simple.